src/matchers/minLengthMatcher.ts

```diff
@@ -10,8 +10,11 @@ export const minLengthMatcher = (minLength: number): Matcher => ({
       return matches;
     }
   },
-  feedback(_match) {
-    return { warning: `Password may not be shorter than ${minLength} characters.`, suggestions: [] };
+  feedback: options => {
+    return {
+      warning: (options.translations.warnings.minLength = 'minLength'),
```

Comment:
maybe define the matcher name on top and use it both here and for the pattern?

or even maybe create an object of all the matcher names to reuse in your own code and also export, so that the consumer of the package would be able to use when adding the matcher. i.e
```js
// you code
export MatcherNames = {
   lowerCaseRequired: 'lowerCaseRequired',
   upperCaseRequired: 'upperCaseRequired',
   ...
}

// usage
import {MatcherNames, lowerCaseRequired, upperCaseRequired } from 'zxcvbn-custom-matchers';

zxcvbnOptions.addMatcher(MatcherNames.lowerCaseRequired, lowerCaseRequired);
zxcvbnOptions.addMatcher(MatcherNames.upperCaseRequired, upperCaseRequired);
...
```

Reply:
I think its less descriptive for the usage example..